frontend/app/events/[id]/ConfigSectionClient.tsx

```
@@ -49,28 +50,40 @@ function ConfigDialog({ title, icon, configHtml, rawConfig, label }: ConfigDialo
+                                size="icon"
+                                className="hover:bg-white/10 text-white/70 hover:text-white shrink-0"
+                            >
+                                <X className="size-4" />
+                                <span className="sr-only">Close</span>
+                            </Button>
+                        </DialogClose>
+                    </div>
                 </div>
+                <div
+                    className="overflow-auto flex-1 px-6 pb-6 pt-4 [&_pre]:!bg-transparent [&_pre]:!p-0 [&_pre]:!m-0 prose dark:prose-invert max-w-none"
+                    dangerouslySetInnerHTML={{ __html: configHtml }}
```

Biome (2.3.13)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)